upgrading/automated_upgrades.adoc

```diff
@@ -591,6 +591,25 @@ Ansible playbook. This is the playbook to use if you were deploying logging for
 the first time on an existing cluster, but is also used to upgrade existing
 logging deployments.
 
+. If you have any Elasticsearch SearchGuard indices in the following naming format, you need to delete and reseed the indices because
+the naming format has changed:
++
+----
+.searchguard.logging-es-*
+----
++
+.. Run the following command to delete the SearchGuard indices:
++
+----
+# oc exec -c elasticsearch $pod -- es_util --query=.searchguard* -XDELETE
```

Comment:
This looks correct. I can only comment if you run it on a system without logging maybe the error is not so obvious.  and `$pod` is the pod name